frontend/src/metabase/static-viz/components/XYChart/utils/margin.ts

```diff
@@ -3,7 +3,7 @@ import { CHART_PADDING } from "metabase/static-viz/components/XYChart/constants"
 import { ChartSettings } from "metabase/static-viz/components/XYChart/types";
 
 export const LABEL_OFFSET = 10;
-export const GOAL_MARGIN = 6;
+export const MARGIN = 6;
```

Comment:
just a rename to indicate that it's not only a goal line that should have margin.